android/src/main/java/com/passagereactnative/PassageReactNativeModule.kt

```diff
@@ -45,10 +48,16 @@ class PassageReactNativeModule(reactContext: ReactApplicationContext) :
   // region PASSKEY METHODS
 
   @ReactMethod
-  fun registerWithPasskey(identifier: String, promise: Promise) {
+  fun registerWithPasskey(identifier: String, optionsMap: ReadableMap?, promise: Promise) {
     CoroutineScope(Dispatchers.IO).launch {
       try {
-        val authResult = passage.registerWithPasskey(identifier)
+        var options: PasskeyCreationOptions? = null
+        optionsMap?.getString("authenticatorAttachment")?.let {
+          AuthenticatorAttachment.decode(it)?.let {
+            options = PasskeyCreationOptions(it)
```

Comment:
this line assumes that 'it' will not be null after decoding. Is that always the case?

Reply:
Line 57 will not be reached if the decoded value is null 👍